cache/refs.go

```diff
@@ -585,7 +587,7 @@ func (sr *immutableRef) Clone() ImmutableRef {
 	return sr.clone()
 }
 
-func (sr *immutableRef) ociDesc(ctx context.Context, dhs DescHandlers) (ocispecs.Descriptor, error) {
+func (sr *immutableRef) ociDesc(ctx context.Context, dhs DescHandlers, convertNonDist bool) (ocispecs.Descriptor, error) {
```

Comment:
`convertNonDist` -> `forceDistributable` . Atm it is hard to understand if this converts into non-dist, or from non-dist.

Reply:
Fixed (used `preferNonDist`)